spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -43,6 +43,8 @@ content into your application. Rather, pick only the properties that you need.
 	logging.file= # Log file name. For instance, `myapp.log`
 	logging.file.max-history= # Maximum of archive log files to keep. Only supported with the default logback setup.
 	logging.file.max-size= # Maximum log file size. Only supported with the default logback setup.
+	logging.filters.console= # Comma-separated list of filter classes to apply to console appender. Only supported with the default logback setup.
+	logging.filters.file= # Comma-separated list of filter classes to apply to file appender. Only supported with the default logback setup.
```

Comment:
I am not a huge fan of comma separated list of FQNs in `application.properties`. I'd assume that it's not something you actually need to change at runtime, is it? How about a programmatic way of doing the same thing?

It can be tricky as the context hasn't been loaded yet though.

Reply:
> I'd assume that it's not something you actually need to change at runtime, is it?

Correct.

> How about a programmatic way of doing the same thing?

That sounds nice, I did consider that at the time I was putting this PR together, however since logging system is initialized very early I didn't see that as a realistic option at the time.